sale_timesheet_project_foresee/README.rst

```diff
@@ -0,0 +1,58 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==========================
+Sale Order Project Foresee
+==========================
```

Comment:
I'm not comfortable with this module name. May be 'sale_generate_project_manually'? Any better idea out there?

Reply:
I would keep the ```sale_timesheet``` prefix because it's the module which provides the task generation. Maybe ```sale_timesheet_project_manual```?